src/main.js

```diff
@@ -35,53 +30,77 @@ process.on('uncaughtException', (exception: Error) => {
  */
 const container = new Container();
 defaultBindings(container, settings);
-
 const deviceServer = container.constitute('DeviceServer');
-deviceServer.start();
-
 const app = createApp(container, settings);
 
-const onServerStartListen = (): void =>
-  console.log(`express server started on port ${NODE_PORT}`);
+const startServers = () => {
+  deviceServer.start();
```

Comment:
OK, that makes sense -- let's see if we can simplify in `spark-protocol` and then come back to this.